app/models/video.rb

```diff
@@ -4,6 +4,7 @@ class Video < ActiveRecord::Base
 
   attr_accessible :description, :title, :youtube_id, :starred, :duration
   acts_as_taggable
+  acts_as_taggable_on :category
```

Comment:
Why didn't we make a table that represents categories? What are the benefits of acts_as_taggable in terms of speed and efficiency? I feel like adding tags is less intuitive and more difficult than adding relations.